lib/hocon/impl/tokenizer.rb

```diff
@@ -280,7 +374,7 @@ def pull_quoted_string
       while c != '"'
         c = next_char_raw
         if c == -1
-          raise self.problem(@line_origin, c, "End of input but string quote was still open", false, nil)
+          raise TokenIterator.problem(@line_origin, c, "End of input but string quote was still open", false, nil)
```

Comment:
I think in these cases you'd generally call `self.class.problem` rather than `TokenIterator.problem`, since this is within the `TokenIterator` class. Not sure if that's conventional or not, though, so it might be good for someone else to way in on that.